ckanext/unaids/plugin.py

```diff
@@ -119,6 +120,19 @@ def can_validate(self, context, data_dict):
         if data_dict.get('schema'):
             return True
 
+    # IPackageController
+    def after_update(self, context, pkg_dict):
+        org_to_allow_transfer_to = [
+            item['value']
+            for item in pkg_dict['extras']
+            if item['key'] == 'org_to_allow_transfer_to'
+        ][0]
```

Comment:
❗ This is causing a bug when uploading resources with

```
Traceback (most recent call last):
  File "/usr/lib/ckan/venv/lib/python2.7/site-packages/flask/app.py", line 2449, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/ckan/venv/lib/python2.7/site-packages/flask/app.py", line 1866, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/ckan/venv/lib/python2.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/ckan/venv/lib/python2.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/ckan/venv/lib/python2.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/ckan/venv/lib/python2.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/ckan/venv/lib/python2.7/site-packages/flask_debugtoolbar/__init__.py", line 125, in dispatch_request
    return view_func(**req.view_args)
  File "/usr/lib/ckan/venv/lib/python2.7/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/usr/lib/ckan/venv/lib/python2.7/site-packages/flask/views.py", line 163, in dispatch_request
    return meth(*args, **kwargs)
  File "/usr/lib/ckan/venv/src/ckan/ckan/views/resource.py", line 248, in post
    get_action(u'resource_create')(context, data)
  File "/usr/lib/ckan/venv/src/ckan/ckan/logic/__init__.py", line 473, in wrapped
    result = _action(context, data_dict, **kw)
  File "/usr/lib/ckan/venv/src/ckan/ckan/logic/action/create.py", line 319, in resource_create
    _get_action('package_update')(context, pkg_dict)
  File "/usr/lib/ckan/venv/src/ckan/ckan/logic/__init__.py", line 473, in wrapped
    result = _action(context, data_dict, **kw)
  File "/usr/lib/ckan/venv/src/ckan/ckan/logic/action/update.py", line 330, in package_update
    item.after_update(context, data)
  File "/usr/lib/adx/ckanext-unaids/ckanext/unaids/plugin.py", line 129, in after_update
    ][0]
IndexError: list index out of range
```

Reply:
Fixed in https://github.com/fjelltopp/ckanext-unaids/pull/92/commits/ad8981aa417930637e53f7394049f335e9ec5e8f